Building on Linux fails

I have assimp dev package installed on Debian Unstable, and yet:

 CMake Error at CMakeLists.txt:11 (include):
   include could not find load file:

     GnuInstallDirs

 CMake Error at irisgl/CMakeLists.txt:20 (add_subdirectory):
   The source directory

     /home/user/path/jahshaka/irisgl/src/assimp

   does not contain a CMakeLists.txt file.

 CMake Error at CMakeLists.txt:296 (set_target_properties):
   set_target_properties Can not find target to add properties to: assimp

That got me further, and then:

/home/user/soft/graphics/jahshaka/irisgl/src/graphics/../vr/../libovr/Include/Extras/OVR_CAPI_Util.h:11:10: fatal error: OVR_CAPI.h: No such file or directory
 #include "OVR_CAPI.h"
          ^~~~~~~~~~~~
compilation terminated.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,6 @@ find_package(Qt5 REQUIRED COMPONENTS Concurrent Widgets Sql Svg)
 
 include(UseModernCpp)
 include(BuildType)
-include(GnuInstallDirs)
 
 # set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTOUIC ON)

--- a/irisgl/CMakeLists.txt
+++ b/irisgl/CMakeLists.txt
@@ -196,7 +196,7 @@ target_include_directories(IrisGL PUBLIC
     "${PROJECT_SOURCE_DIR}/src/"
     "${PROJECT_BINARY_DIR}/src/assimp/include/"
     "${PROJECT_SOURCE_DIR}/src/assimp/include/"
-    "${PROJECT_SOURCE_DIR}/src/libovr/include"
+    "${PROJECT_SOURCE_DIR}/src/libovr/Include"
 )
 
 target_link_libraries (IrisGL assimp Qt5::Core Qt5::OpenGL)

